include/roboteam_ai/control/ControlModule.h

```diff
@@ -0,0 +1,61 @@
+//
+// Created by jaro on 15-10-20.
+//
+
+#ifndef RTT_CONTROLMODULE_H
+#define RTT_CONTROLMODULE_H
+
+#include <roboteam_proto/RobotCommand.pb.h>
+
+#include "stp/StpInfo.h"
+#include "world/views/RobotView.hpp"
+
+namespace rtt::ai::control {
+
+/**
+ * Control module as seen in the system architecture.
+ * Its job: Receive RobotCommands from Skills, check these and limit and/or change things wherever necessary.
+ * For now, it limits the velocity and acceleration that a Skill sends so the robot doesn't do a backflip.
+ * And it checks if the command that was sent makes sense (e.g., no kicking and chipping at the same time.
+ */
```

Comment:
Minor, but use the java-style documentation: (@brief, @author) for classes. That way your CLion can display it when you hover over the class :). I'm happy you wrote some documentation, by the way :D